### DISCUSSION TITLE
Add options for 1/3 and 1/4 to Top or Bottom

### DISCUSSION THREAD

**Discussion by mauriciolima1988** [rxhanson/Rectangle#1262] [Ideas]
Along with a 34" monitor, I use two vertical monitors, it would be very useful if we could resize windows to 1/3 and 1/4 to Top or Bottom of the screen (I don't know Swift to make this happen).

**Comment by DanielWinks** [comment]
I've got an LG DualUp (which is honestly a game-changer for development work). One feature I would love to see is the ability to do a 1/3 : 2/3 split top/bottom. On this monitor, and other vertical arrangements, it's often desirable to have the main work area the bottom 2/3 and the top 1/3 used for documentation, PostMan, etc.

Or 2/3 : 1/3 top/bottom, depending on what I'm doing.

Right now there's 1/2 : 1/3, but there's a lot of times where I would like to devote a bit more space to one and a little less to the other, so 1/3 : 2/3 and 2/3 : 1/3 option would be really appreciated.

**Comment by rxhanson** [comment]
Rectangle will automatically do thirds and fourths oriented vertical (portrait ratio) displays. Are you looking for something different than what's currently there? Maybe an image would help me understand a little better?

**Reply by DanielWinks** [reply]
I see the automatic thing, though it's a bit hard to trigger before it scrolls over to the next display.

I didn't see anything for creating a 1/3 or 2/3 hotkey, or anything in these menus:
<img width="228" alt="image" src="https://github.com/rxhanson/Rectangle/assets/5339479/f82f7066-d604-488b-bb94-bbb35edab9b9">
![image](https://github.com/rxhanson/Rectangle/assets/5339479/786ce832-74f8-467b-bb07-ec730f680514)

I see options for 1/3 and 2/3 on those menus for landscape, but not portrait.

**Reply by DanielWinks** [reply]
Looks like this option is what I needed: 
![image](https://github.com/rxhanson/Rectangle/assets/5339479/bd09a9c5-6db4-4362-9640-15181203ff73)

That lets me toggle through 1/3, 1/2, and 2/3 via command. That's more than sufficient.

This is by far the best window manager I've ever used, not only on Mac, but all platforms.

**Reply by rxhanson** [reply]
If you execute those window actions for thirds and you are on a portrait ratio display, then you should get portrait thirds, regardless of icon in the menu. I take it you're getting a third that is on the left side of the screen, not on the top for "first third"?

**Reply by rxhanson** [reply]
Ok, sounds good, glad you have something working :)

**Reply by DanielWinks** [reply]
Yep, it does just that with the above option turned on. I was looking for somewhere to specify hotkeys for 1/3 and 2/3, but with the above option on I don't need them, control-option-up/down cycles through them. Very nice.
